### PR TITLE
Die Rechnung bitte

### PR DESCRIPTION
**Pull request by saantiaguilera**
- Agregamos categorizacion en base a intervalos para poder tener un output splatteado en 5
- Mini cambios en el modelo (mas que nada en las metrics)
- Agrego loss al final para poder verla (Como no se nos brinda un csv de test, hacemos el ""test"" como si fuese una epoch 201)
- Grafico accuracy y loss para que se vea como se fue entrenando

### CODE REVIEW

**Comment by aibarbetta** [comment]
No hay que mappear los resultados de la prediccion a las clases y despues hacer un accuracy score?
```
# % de casos de éxito 
from sklearn.metrics import accuracy_score
accuracy = accuracy_score(classified_y_test, classified_predictions)
print('Evaluated accuracy: ', (accuracy * 100))
```

fijate que en el enunciado dice que para la de clasificacion hay que decirle el % de casos de exito, no el error cuadratico medio

**Comment by saantiaguilera** [comment]
Esta como parte del grafico y en la ultima epoch (39%) pero lo agrego por separado
